File: LV1/1.3/fizzbuzz/fizzbuzz.rb
def fizzbuzz(n, x, y)
  mx = (n % x).zero?
  my = (n % y).zero?

  case
    when (mx && my) then 'FizzBuzz'
    when mx then 'Fizz'
    when my then 'Buzz'
    else n
  end
end

x, y, n = gets.split.map(&:to_i)

(1..n).each { |i| puts fizzbuzz(i, x, y) }

File: LV1/1.3/fizzbuzz/fizzbuzz.py
def fizzbuzz(n, x, y):
    mx = n % x == 0
    my = n % y == 0

    if mx and my:
        return "FizzBuzz"
    elif mx:
        return "Fizz"
    elif my:
        return "Buzz"
    else:
        return n


x, y, n = [int(i) for i in input().split(" ")]

[print(fizzbuzz(i, x, y)) for i in range(1, n+1)]
